ports/atmel-samd/mpconfigport.h

```diff
@@ -112,6 +112,11 @@
 #define CIRCUITPY_DEFAULT_STACK_SIZE                4096
 #endif
 
+#ifndef BOARD_BROWNOUT_LEVEL (
```

Comment:
stray `(` here.

Also copy the comment about why the level is 2.7V. Any board with external SPI flash chips like the ones we use will need 2.7V min due to the requirements of those chips.